### PR TITLE
Removes clan sword parry

### PR DESCRIPTION
**Pull request by Joelampost**
## About The Pull Request

Predators are not jedi them being able to block super sonic bullets with a blade is LRP

## Why It's Good For The Game

It removes a win button from preds
This PR buffs the sharpness of the sword because it was as sharp as wire cutters before 

## Changelog

:cl: Joe Lampost
add: buffs the sword sharpness
del: Removed clan sword parry
/:cl:

### CODE REVIEW

**Review by Shad0vvs — CHANGES_REQUESTED** [review]
You removed the X.interference from the weapon line 197-199.

**Comment by DylonBR** [comment]
entirety of the predcord coming to downvote this soon

**Comment by carlarctg** [comment]
as the guy who added parry i approve

File: code/modules/cm_preds/yaut_weapons.dm [hunk 1]
```diff
@@ -181,134 +181,12 @@
...
-
-
-/obj/item/weapon/melee/yautja/sword/attack(mob/living/target, mob/living/carbon/human/user, var/riposte)
-	if(parrying && !riposte)
-		to_chat(user, SPAN_WARNING("You're a bit busy concentrating to hit something."))
-		return
-	. = ..()
-	if(!.)
-		return
+/obj/item/weapon/melee/yautja/sword/attack(mob/living/target as mob, mob/living/carbon/human/user as mob)
```
Comment (carlarctg) [inline code/modules/cm_preds/yaut_weapons.dm:184]: you don't need this line, just remove the proc entirely

Comment (carlarctg) [reply code/modules/cm_preds/yaut_weapons.dm]: This Is Now Fixeder

**Review by stanalbatross — APPROVED** [review]

**Review by stanalbatross — APPROVED** [review]
I will architect approve this because the clan sword parry is a quite unengaging and unforgiving mechanic. It's mainly used to cheese during fights and bait your opponent into being overly cautious when fighting you or they end up shooting themselves. I honestly can't say that it's balanced either, considering it allows preds to basically 1-hit their opponents via reflected PBs as well as make themselves invulnerable to gunfire.

**Pull request merged**